sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,74 @@
+import atexit
+from queue import Empty,Queue
+from rllab.sampler.utils import rollout
+import numpy as np
+from threading import Thread
+import tensorflow as tf
+
+__all__ = [
+    'init_worker',
+    'init_plot',
+    'update_plot',
+    'shutdown_worker'
+]
+
+process = None
+queue = None
+
+class PlotterThread(Thread):
+    def __init__(self, queue,sess):
```

Comment:
Please format all new code according to PEP8 using YAPF.

Reply:
Done